src/query/interpreter.hpp

```diff
@@ -247,6 +248,14 @@ class Interpreter final {
     std::optional<std::string> db;
   };
 
+#ifdef MG_ENTERPRISE
+  struct RouteResult {
+    int ttl{300};
+    std::string db{};  // Currently not used since we don't have any specific replication groups etc.
```

Comment:
Can you explain this part, when what do replication groups mean here?

Reply:
This uses Neo4j for routing between replication groups. We don't support this atm